.eslintrc.json

```diff
@@ -50,7 +50,8 @@
           ]
         }
       }
-    ]
+    ],
+    "no-restricted-exports": ["error", { "restrictDefaultExports": { "direct": true } }]
```

Comment:
@skochay330 this change is already present on the main branch. There are others changes added here that already present on the main branch. Do you intentionally add those again?

Reply:
rebasing after adding a merge commit is almost impossible :)

I'd suggest starting a new branch from main, and just adding all the intentionally changed files